### ISSUE TITLE
Add IPFS as a host target

### ISSUE THREAD

**Issue by lsaether** [w3f/polkadot-wiki#369]
Alongside GH pages, we should start hosting the wiki on IPFS. The best way to do this would probably be by adding a CI job or GitHub action that triggers when the GH pages branch is built. This job would upload the built files onto IPFS and pin it.

cc @swader

**Comment by Swader** [comment]
We *could* partner with providers like Fleek.co for this. They support auto-deploy-and-pin-to-ipfs from standard build pipelines. @fgimenez would this be doable? The free plan would work I think. https://fleek.co/pricing/

**Comment by fgimenez** [comment]
Absolutely, IPFS in general would make a lot of sense. 

fleek looks fine but i think we don't need an external service, we could just run our own pinning node and on CI build the site, compute the CID, pin the content and update a DNSLink to that content https://docs.ipfs.io/guides/concepts/dnslink/ wdyt? 

We could do some testing and publish to a test domain in parallel to the regular build.

**Comment by Swader** [comment]
I figured Fleek for simplicity but if it's as easy for you as you say, please, that'd be great!

**Comment by lsaether** [comment]
Sites are now being hosted on IPFS but we need to add DNSLink before they are easily accessible. Opened a new issue for that (#696) and closing this one.
